app/(afterLogin)/(timesheetInformation)/timesheet/settings/approvals/_component/approvalFilter/index.tsx

```
@@ -38,12 +38,14 @@ const ApprovalFilter = () => {
     setApproverType('');
   };
   return (
-    <div className="flex sm:block">
+    <div className="flex justify-between gap-4 sm:block">
+      <div className="flex-1">
       <ApprovalFilterComponent
```

ESLint

[error] Insert `··`

(prettier/prettier)